pages/index.js

```diff
@@ -122,6 +134,13 @@ class IndexPage extends Component {
             }
           `}
         </style>
+        <style jsx global>
+          {`
+            body {
+              font-family: helvetica;
```

Comment:
So i stumbled upon this article:
https://css-tricks.com/sans-serif/
And it says we should just use san-serif declaration here. like so:
```suggestion
              font-family: Sans-Serif;
```
It's a good thing i did, because were it up to me i would have tone exactly what you did :)